server/src/main/java/io/littlehorse/common/model/getable/core/taskrun/TaskRunModel.java

```diff
@@ -204,8 +204,9 @@ public void processStart(TaskClaimEvent se) {
         taskResult.setTaskRunId(id);
         taskResult.setTime(new Date(System.currentTimeMillis() + (1000 * timeoutSeconds)));
         taskResult.setStatus(TaskStatus.TASK_TIMEOUT);
-
-        LHTimer timer = new LHTimer(new CommandModel(taskResult, taskResult.getTime()), getDao());
+        CommandModel timerCommand = new CommandModel(taskResult, taskResult.getTime());
```

Comment:
Please make the `tenantId` a required argument for the `CommandModel` constructor so that:

1. It is impossible to have a null tenant id.
1. This code can be one line again